debian/copyright

```diff
@@ -0,0 +1,9 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ev3dev-lang-python
```

Comment:
FYI: you are having Source: set for e.g. brickman:
https://github.com/ev3dev/brickman/blob/ev3dev-jessie/debian/copyright#L3
but not in others. So I removed them too.